test/src/main/java/org/apache/accumulo/test/SampleIT.java

```diff
@@ -490,7 +490,7 @@ private void assertSampleNotPresent(SamplerConfiguration sc, ScannerBase... scan
 
       scanner.clearSamplerConfiguration();
       for (Entry<Key,Value> entry : scanner) {
-        entry.getKey();
+        var unusedRetVal = entry.getKey();
```

Comment:
The call to `.getKey` was only ever added to avoid a previous warning about the loop variable being unused. It would probably be sufficient to have an assert that the loop variable is non-null.

Reply:
Wrapped calls to entry.getKey() with assertNotNull in SampleIT, TImeoutIT, and ServerSideErrorIT.